modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -3575,37 +3496,6 @@ a|A path.
 m|+++/var/run/secrets/kubernetes.io/serviceaccount/namespace+++
 |===
 
-
-[role=label--enterprise-edition label--deprecated-5.23]
-[[config_dbms.kubernetes.service_port_name]]
-=== `dbms.kubernetes.service_port_name`
-
-.dbms.kubernetes.service_port_name
-[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
-|===
-|Description
-a|Service port name for discovery for Kubernetes API.
-|Valid values
-a|A string.
-|Default value
-m|
-|===
-
-[role=label--enterprise-edition]
-[[config_dbms.kubernetes.discovery.v2.service_port_name]]
-=== `dbms.kubernetes.discovery.v2.service_port_name`
-
-.dbms.kubernetes.discovery.v2.service_port_name
```

Comment:
I think this needs to be converted to `dbms.kubernetes.discovery.service_port_name` rather than removed.

Reply:
I see I said to remove it earlier, I think I was just plain wrong... Sorry.